### DISCUSSION TITLE
How to fix Error code 403

### DISCUSSION THREAD

**Discussion by zulbil** [hasura/graphql-engine#9348] [General]
Hey guys,

I'm trying to execute a query from my local machine. But i keep having the error code 403 forbidden.
I include the header in the request and everything required. I'm using graphql-request library.
Here is how my request is formatted :
```
const graphQLClient = new GraphQLClient(endpoint, {
mode: 'cors',
credentials: 'include',
headers: {
'content-type': 'application/json',
'x-hasura-admin-secret': adminSecret
}
});
```
Any help will be really appreciated

**Comment by meetzaveri** [comment]
Hey @zulbil 

We received a support ticket from your end, and we replied to that ticket.

Although, we wanted to know if you had configured the CORS according to the docs here - https://hasura.io/docs/latest/deployment/graphql-engine-flags/config-examples/#configure-cors

Are you still facing any issues ?

**Comment by zulbil** [comment]
Hey @meetzaveri 
I did configure the CORS as mentioned in the doc. But I keep facing the same issue

**Comment by meetzaveri** [comment]
Can you try setting up a sample app in codesandbox.io with the example request you provided? So that we can collaborate and troubleshoot together that what's causing the issue. We would also like to know about the configuration in client.
